### ISSUE TITLE
SC2050: false positive: "This expression is constant"

### ISSUE THREAD

**Issue by bittorf** [koalaman/shellcheck#506]
```shell
#!/bin/sh

[ "file1" -ot "file2" ]
test "file1" -ot "file2"
```

the first check triggers, the second not. (git-checkout @ 21 o'clock)

**Comment by koalaman** [comment]
Very nice catch. Fixed in 8d9d453!

**Comment by bittorf** [comment]
tested 630f20e888d4e2f14dc6051847067cc516345fec - and it's OK now - thank you!
